addon/install.rdf

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<RDF xmlns="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+     xmlns:em="http://www.mozilla.org/2004/em-rdf#">
+  <Description about="urn:mozilla:install-manifest">
+    <em:id>pageshot@mozilla.org</em:id>
+    <em:name>Page Shot</em:name>
+    <em:targetApplication>
+      <Description>
+        <em:id>{ec8030f7-c20a-464f-9b0e-13a3a9e97384}</em:id> <!--Firefox-->
+        <em:minVersion>51.0a1</em:minVersion>
```

Comment:
I haven't tested on 51, but that's when Embedded Extension support landed.  I have tested 53 and it works, we have a fallback for the one 54-only API that we use.  OTOH, it's probably safer for pre-54 people to use the WebExtension and not the bootstrap extension.

Note ESR is Firefox 52, so there's some value is supporting 52.